src/forms/citizenship.js

```diff
@@ -1,5 +1,5 @@
 // REACT COMPONENTS
-import React, { Component } from 'react';
+import React from 'react';
```

Comment:
It may be a good idea to add a note about this in the commit message, since removing excess imports is something different than removing nested props. If someone needs to go back to hunt for when this was done, it'd be helpful to be able to see it at least one commit message.